src/content/reference/react/useTransition.md

```diff
@@ -16,11 +16,11 @@ const [isPending, startTransition] = useTransition()
 
 ---
 
-## Reference {/*reference*/}
+## Referensi {/*reference*/}
 
 ### `useTransition()` {/*usetransition*/}
 
-Call `useTransition` at the top level of your component to mark some state updates as transitions.
+Panggil `useTransition` pada level teratas komponen Anda untuk menandai beberapa perubahan state sebagai transisi.
```

Comment:
Same with this suggestion https://github.com/reactjs/id.react.dev/pull/371/files#r1180040206